frontend/package.json

```diff
@@ -100,6 +100,7 @@
     "html-webpack-plugin": "^5.5.0",
     "mini-css-extract-plugin": "^2.6.0",
     "prettier": "^2.5.1",
+    "sass": "^1.53.0",
```

Comment:
just curious what's this needed for?

Reply:
because `node-sass` lib was depreciated and we can’t use on machines with M1 chip, so when use `sass` everything works 😊